p/scripts/main.js

```diff
@@ -1215,6 +1215,84 @@ function init_notifications() {
 }
 // </notification>
 
+// <popup>
+var popup = null,
+	popup_iframe = null,
+	popup_p = null,
+	popup_working = false;
+
+function openPopupWithMessage(msg) {
+	if (popup_working === true) {
+		return false;
+	}
+
+	popup_working = true;
+
+	popup_p.innerHTML = msg;
+	popup_p.removeAttribute('hidden');
+
+	popup.style.display = "block";
```

Comment:
Not sure why you're changing quoting styles back & forth but I find it really distracting from the actual logic. ;-)
```suggestion
	popup.style.display = 'block';
```

Reply:
Mainly because I googled a lot about how to create pop-up. Plus lake of habit to play with different manner to declare a string ^^'

I will fix all of them.